packages/eslint-config-yc-react/README.md

```diff
@@ -34,4 +34,8 @@ It inherits some basic packages like:
 
 ## Using the .prettierrc.js in your project
 
-You can use the same config as yc-base has [here](../eslint-config-yc-base/README.md#using-the-.prettierrc.js-in-your-project)
+In your project you can symlink to the yc-base prettierrc using the following command:
+
+```ln -s node_modules/eslint-config-yc-base/.prettierrc.js ./.prettierrc.js```
```

Comment:
do we need to add the `@youngcapital` namespace to this path?

Reply:
ah yes! Good one!